stylus/settings/breakpoints.styl

```diff
@@ -22,21 +22,35 @@ breakpoints = {
     'medium': 'm',
 }
 
+/*
+    Breakpoints
+
+    This is the collection of available breakpoint variables:<br><br>
+    - `BP-teeny`         -  375px
+    - `BP-tiny`          -  480px
+    - `BP-small`         -  768px
+    - `BP-medium`        -  1023px
+    - `BP-large`         -  1200px
+    - `BP-extra-large`   -  1439px
+
+    <br>Also we have a standard breakpoints collection meant to be used as suffixes for some utilities<br>
+
+    If you want a utility class to trigger only at defined breakpoints you can use those:<br><br>
```

Comment:
```suggestion
    If you want a utility class to trigger only at a defined breakpoints you can use those:<br><br>
```

Reply:
Are you sure? I wrote "breakpoints", plural. You shouldn't add "a", should you?

Reply:
No you're completely right, sorry.